pyccel/ast/builtins.py

```diff
@@ -613,9 +613,13 @@ class PythonMax(PyccelInternalFunction):
     _shape = ()
     _order = None
 
-    def __init__(self, x):
-        if not isinstance(x, (list, tuple, PythonTuple, PythonList)):
+    def __init__(self, *x):
+        if isinstance(x, (list, tuple)):
+            x = PythonTuple(*x)
+        elif not isinstance(x, (PythonTuple, PythonList)):
             raise TypeError('Unknown type of  %s.' % type(x))
+        if not x.is_homogeneous:
+            raise NotImplementedError("Cannot determine dtype of min({})".format(x))
```

Comment:
errata, max function instead of min
`"Cannot determine dtype of max({})"`

Reply:
Fixed in f8bb4b3